### PR TITLE
Upgrade to python3

### PR DESCRIPTION
**Pull request by pankon**

### CODE REVIEW

File: bplist/bplist.py [hunk 1]
```diff
@@ -271,3 +272,17 @@ def unplist(s):
     from Foundation import NSData, NSPropertyListSerialization
     d = NSData.dataWithBytes_length_(s, len(s))
     return NSPropertyListSerialization.propertyListWithData_options_format_error_(d, 0, None, None)
+
+if __name__ == "__main__":
```
Comment (farcaller) [inline bplist/bplist.py:276]: I guess that adds a nice `python -m bplist` use case. Could you update the docs accordingly, please?

File: bplist/bplist.py [hunk 2]
```diff
@@ -271,3 +272,17 @@ def unplist(s):
...
+    import json
+    file_path = sys.argv[1]
+
+    with open(file_path, "rb") as fp:
+        data = fp.read()
+
+    out = BPListReader(data).parse()
+
+    with open(file_path + ".json", "w") as fp:
+        json.dump(out, indent=4)
```
Comment (farcaller) [inline bplist/bplist.py:288]: Missing a new line in the end of the file.

Comment (pankon) [reply bplist/bplist.py]: Thanks! A great feature would be to add nested decoding
Great library!

**Comment by pankon** [comment]
@farcaller Fixed everything you suggested. Thanks again!

**Pull request merged**